client/tests/e2e/specs/test_active_users_page.ts

```diff
@@ -91,70 +91,85 @@ describe('Check active users page', () => {
-    cy.get('.question-modal').find('#next-button').click();
-    // cspell:disable-next-line
-    cy.checkToastMessage('success', 'Cernd has been revoked and can no longer access this organization!');
-  });
+    cy.get('@userItems').eq(1).realHover().find('.checkbox').should('be.visible');
 
-  it('Tests revoke one selected user', () => {
-    cy.get('.users-container').find('.user-list-item').as('userItems').should('have.length', 3);
-    // Since the checkbox appears only on hover, it's easier to select all and deselect the one we don't want
-    cy.get('.user-list-header').find('ion-checkbox').click();
-    cy.get('@userItems').eq(2).find('ion-checkbox').click();
-    cy.get('@userItems').eq(1).find('ion-checkbox').should('have.class', 'checkbox-checked');
-    cy.get('@userItems').eq(2).find('ion-checkbox').should('not.have.class', 'checkbox-checked');
-    cy.get('.counter').contains('One user selected');
-    cy.get('.contextual-menu').find('#button-revoke-user').contains('Revoke this user').click();
-    cy.get('.question-modal').find('.ms-modal-header__title').contains('Revoke this user?');
-    cy.get('.question-modal').find('#next-button').click();
-    // cspell:disable-next-line
-    cy.checkToastMessage('success', 'Cernd has been revoked and can no longer access this organization!');
-    // Ultimately, they probably should not appear anymore
-    cy.get('@userItems').eq(1).find('ion-checkbox').should('not.have.class', 'checkbox-checked');
-  });
+    cy.get('#activate-users-ms-action-bar').find('#button-invite-user').should('exist');
 
-  it('Tests revoke multiple', () => {
-    cy.get('.users-container').find('.user-list-item').as('userItems').should('have.length', 3);
-    cy.get('.user-list-header').find('ion-checkbox').click();
-    cy.get('@userItems').eq(1).find('ion-checkbox').should('have.class', 'checkbox-checked');
-    cy.get('@userItems').eq(2).find('ion-checkbox').should('have.class', 'checkbox-checked');
-    cy.get('.contextual-menu').find('#button-revoke-user').contains('Revoke these users').click();
-    cy.get('.question-modal').find('.ms-modal-header__title').contains('Revoke these users?');
-    cy.get('.question-modal').find('#next-button').click();
-    // cspell:disable-next-line
-    cy.checkToastMessage('success', '2 users have been revoked and can no longer access this organization!');
-    // Ultimately, they probably should not appear anymore
-    cy.get('@userItems').eq(1).find('ion-checkbox').should('not.have.class', 'checkbox-checked');
-    cy.get('@userItems').eq(2).find('ion-checkbox').should('not.have.class', 'checkbox-checked');
+    cy.get('@userItems').eq(2).realHover().find('.checkbox').click();
+    cy.get('#activate-users-ms-action-bar').find('#button-revoke-user').contains('Revoke this user');
+    cy.get('#activate-users-ms-action-bar').find('#button-common-workspaces').contains('See common workspaces');
   });
+
+  // it('Tests context menu', () => {
```

Comment:
Did you forget to uncomment those?

Reply:
Yes, I was to accelerate the test, I uncommented them.